### PR TITLE
Added focchi UI-Configs for dmev2

### PR DESCRIPTION
**Pull request by AleLuc98**
Added focchi UI-Configs for dmev2

### CODE REVIEW

File: mf-focchi/common/services.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+[
+   {
+      "img": "//mescobrad.digital-enabler.eng.it/suite/static/imgs/icons/icon_",
```
Comment (robraccu) [inline mf-focchi/common/services.json:3]: mascobrad ?

File: mf-focchi/config.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+   "realm": "MESCOBRAD",
```
Comment (robraccu) [inline mf-focchi/config.json:2]: I guess FOCCHI ;-)

File: mf-focchi/dme/dme-auth-config.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+   "mf": "Auth",
+   "api": "https://dme-api.digital-enabler.eng.it/api",
+   "keycloak": {
+      "url": "https://idm.digital-enabler.eng.it/auth",
+      "clientId": "mescobrad",
```
Comment (robraccu) [inline mf-focchi/dme/dme-auth-config.json:6]: I guess focchi

File: mf-focchi/dme/dme-import-map.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "@digital-enabler/mashups-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/mashups-list@1.0.0/app.js",
+    "@digital-enabler/mashups-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/mashups-editor@1.0.1/app.js",
+    "@digital-enabler/mashups-editor-appbar": "https://cdn.jsdelivr.net/npm/@digital-enabler/mashups-editor-appbar@1.0.2/app.js",
+    "@digital-enabler/share-form": "https://cdn.jsdelivr.net/npm/@digital-enabler/share-form@1.0.0/app.js",
+    "@digital-enabler/dme-datasources-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-datasources-list@1.0.0/app.js",
+    "@digital-enabler/dme-datasource-details": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-datasource-details@0.0.11/app.js",
+    "@digital-enabler/dme-flows-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flows-list@1.0.0/app.js",
+    "@digital-enabler/dme-flow-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flow-editor@1.0.1/app.js",
+    "@digital-enabler/dme-gui": "https://mashups.mescobrad.digital-enabler.eng.it/demf-dme-gui.js"
```
Comment (robraccu) [inline mf-focchi/dme/dme-import-map.json:20]: I guess focchi

File: mf-focchi/dme/dme-sidebar-config.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{
+   "mf": "Sidebar",
+   "api": "https://dme-api.digital-enabler.eng.it/api",
+   "storageImgs": "https://storage.digital-enabler.eng.it/imgs",
+   "jsonTools": "https://raw.githubusercontent.com/digital-enabler/UI-Configs/main/mf-mescobrad/common/services.json",
```
Comment (robraccu) [inline mf-focchi/dme/dme-sidebar-config.json:5]: I guess focchi

File: mf-mescobrad/common/services.json [hunk 6]
```diff
@@ -1,24 +1,6 @@
...
-      "name": "Database Manager",
-      "link": "",
-      "prefix": "database.",
-      "path": "",
-      "active": false,
-      "category": "data_processing"
-   },
-   {
-      "img": "//mescobrad.digital-enabler.eng.it/suite/static/imgs/icons/icon_",
+      "img": "//focchi.digital-enabler.eng.it/suite/static/imgs/icons/icon_",
```
Comment (robraccu) [inline mf-mescobrad/common/services.json:3]: it seems you inverted files....mescobrad with focchi

**Review by robraccu — COMMENTED** [review]
There should be a lot of comments but I stopped, at first glance it looks like you have swapped the focchi files with the mescobrad files...

**Review by robraccu — APPROVED** [review]

**Review by sirnino — APPROVED** [review]

**Pull request merged**